server/src/auth/auth.controller.ts

```diff
@@ -0,0 +1,90 @@
+import { Response } from 'express'
+import { User, Prisma } from '@prisma/client'
+
+import {
+    Body,
+    Controller,
+    Get,
+    HttpCode,
+    HttpException,
+    HttpStatus,
+    Logger,
+    Post,
+    Res,
+    Req,
+    UseGuards,
+} from '@nestjs/common'
+
+import { AuthService } from './auth.service'
+import { RequestWithUser } from './dto'
+import { JwtAuthGuard, JwtAuthLocalGuard } from './guards'
+import { PostgresErrorCode } from '../../database/postgresErrorCodes.enum'
+import { UserService } from '../user/user.service'
+@Controller('auth')
+export class AuthController {
+    private readonly logger: Logger
```

Comment:
I prefer to using [this way of logging](https://docs.nestjs.com/techniques/logger#using-the-logger-for-application-logging)